client/ayon_houdini/api/plugin.py

```diff
@@ -334,6 +348,34 @@ class HoudiniInstancePlugin(pyblish.api.InstancePlugin):
     hosts = ["houdini"]
     settings_category = SETTINGS_CATEGORY
 
+    def evalParmNoFrame(self, rop, parm, **kwargs):
```

Comment:
We should do a follow up PR potentially to refactor this to lower case, `eval_parm_no_frame` to adhere to the AYON code style?

Reply:
Should I do it in this PR instead?

Reply:
created an issue for it
https://github.com/ynput/ayon-houdini/issues/191